python/cudf/cudf/_lib/column.pyx

```diff
@@ -428,6 +438,47 @@ cdef class Column:
             offset,
             children)
 
+    cpdef pylibcudf.Column to_pylibcudf(self):
+        # TODO: Categoricals will need to be treated differently eventually.
+        # There is no 1-1 correspondence between cudf and libcudf for
+        # categoricals due to the ordering question.
```

Comment:
Maybe reference some comment on the "ordering question" for those of us (like me) how don't know what it is.

Reply:
Added more of a note.